msyncd/AccountsHelper.cpp

```diff
@@ -42,7 +44,8 @@ AccountsHelper::AccountsHelper(ProfileManager &aProfileManager, QObject *aParent
     QObject::connect(iAccountManager, SIGNAL(accountUpdated(Accounts::AccountId)),
                      this, SLOT(slotAccountUpdated(Accounts::AccountId)));
 
-    registerAccountListeners();
+    // load accounts after return from contructor
```

Comment:
The signal "enableSOC" is fired on "void AccountsHelper::registerAccountListener(Accounts::AccountId id)" and we need to connect to this signal.